teps/0098-workflows.md

```diff
@@ -125,62 +54,105 @@ simplify the Tekton experience. It can provide a single place to declare all the
 
 ### Goals
 
-<!--
-List the specific goals of the TEP.  What is it trying to achieve?  How will we
-know that this has succeeded?
--->
-
 * Provide a way to describe entire CI/CD configuration (not just pipeline definition) in a single place
 * Make managing Tekton configuration less verbose and repetitive.
-* Support end to end CI workflows from source control
 * Make it easier and simpler for end users to get started with a pure Tekton CI system
-* Allow clear separation of concerns between platform developers and app teams
-* Allow for clear separation between run configuration and pipeline definition?
+* Allow clear separation of concerns between platform developers and app teams,
+including making it easier to separate notifications and other infrastructure out of the Pipelines being run
+* Create a Workflows conformance spec
 
 ### Non-Goals
 
-<!--
-What is out of scope for this TEP?  Listing non-goals helps to focus discussion
-and make progress.
--->
 * Non CI/CD pipelines
-* Installation/deployment of other Tekton projects that Workflows needs as pre-requsities
+* Installation/deployment of other Tekton projects that Workflows needs on behalf of the cluster operator
+  * We can explore using the Operator project to simplify installation, but for an initial implementation
+  the cluster operator will just be directed to install the necessary projects
+* In-tree support for all possible SCMs, cloud storage systems, secrets storage systems, etc.
 
 ### Use Cases
 
-<!--
-Describe the concrete improvement specific groups of users will see if the
-Motivations in this doc result in a fix or feature.
+The following example use cases are targeted by the Workflows project, but may be implemented via several milestones
+(i.e. not all of these features need to be supported in an initial implementation). These use cases are examples, not a comprehensive list of features;
+for example, we want to support CI in multiple SCMs, not just Github.
 
-Consider both the user's role (are they a Task author? Catalog Task user?
-Cluster Admin? etc...) and experience (what workflows or actions are enhanced
-if this problem is solved?).
--->
+1. Github-based CI
 
-1. End users can set up an entire end to end CI pipeline (from source back to run results) using only Tekton 
-1. End users can visualize runs grouped by repository
-1. End users can create an end to end CI/CD workflow e.g. no scripts to set up webhooks, GH Apps etc.
+- The end user would like to trigger a CI PipelineRun whenever a pull request is opened or a commit is pushed to the pull request,
```

Comment:
I'm trying to standardize on some common role names for use cases like this: Application Developer, DevOps Engineer, Security & Policy Enforcer (these actually come from the Google UX research team). So here we're talking about the Application Developer, right?

Reply:
This is based on our Tekton user profiles: https://github.com/tektoncd/community/blob/main/user-profiles.md (although "end user" isn't one of them, we're probably targeting "pipeline authors" and "pipeline users)
Would it make sense to update these user profiles to reflect these more granular roles?